### ISSUE TITLE
Memory usage grows indefinitely

### ISSUE THREAD

**Issue by sciurus** [mozilla/classify-client#65]
To reproduce, run the following in three separate shells

```
docker run -p 8000:8000/tcp mozilla/classify-client:latest
```

```
docker stats
```

```
ab -c 1 -n 100000 -k http://localhost:8000/api/v1/classify_client/
```

Observe that memory usage starts at 5MiB but grows continuously, exceeding 600 MiB by the time all 100k requests have been served.

**Comment by sciurus** [comment]
Attached is a zip archive containing the output from running a debug build through `valgrind --tool=massif`, plus the result of running that through `ms_print`,

[massif.zip](https://github.com/mozilla/classify-client/files/3611909/massif.zip)

**Comment by sciurus** [comment]
If I change `ab` to hit the much simpler `http://localhost:8000/__lbheartbeat__` instead of the `classify_client` endpoint, I see the same memory growth.

**Comment by mythmon** [comment]
This was a problem with the Sentry middleware. I've filed #66 to fix that problem, however now we're running into problems with unrelated dependencies, so we can't merge the PR.

**Comment by sciurus** [comment]
I've verified that with the change in #66 memory usage remains at 6MiB as the number of requests handled increases.

**Comment by sciurus** [comment]
This is closed as part of #67
